glue/cirq/stimcirq/_cirq_to_stim_test.py

```diff
@@ -336,10 +336,13 @@ def test_on_loop():
                 cirq.measure(b, key="b"),
             ),
             repetitions=3,
-        )
```

Comment:
Don't pile additional checks onto existing tests. This makes tests incomprehensible. Make a separate test.

Reply:
Done!